src/effects/browse.js

```diff
@@ -12,3 +12,13 @@ export function fetchSchools() {
       querySnapshot => querySnapshot.docs.map(doc => doc.data()),
     );
 }
+
+export function fetchSubjects(termId, subjectId) {
+  return currentTermDoc
+    .collection('subjects')
```

Comment:
You're gonna need to still make sure the subject is associated with the correct school as well, since some subjects can be in multiple schools.